[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: start_1_node
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'first_node', '-p', '42339']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-16T00:28:08.246Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -name first_node
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -port 42339
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-16T00:28:08.246Z	INFO	first_node	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:10.452] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - ERROR[01-16|00:28:10.456] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:10.456] New local node record                    seq=1,736,987,290,455 id=ef085429cb230b52 ip=127.0.0.1 udp=0 tcp=43883
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:10.456] Started P2P networking                   self="enode://ec7c09df0046013a54a421068e4d9c18721bcf89a5b2e6d4729da81432220aac0da6bbaf021675cbe46fb4986f290c4e1807621ea083cb1737deabf79c5148f6@127.0.0.1:43883?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:10.457] HTTP server started                      endpoint=127.0.0.1:42339 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:10.480] New local node record                    seq=1,736,987,290,461 id=9bd3899c0ab5976c ip=10.1.0.12 udp=0 tcp=45153
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:10.481] New local node record                    seq=1,736,987,290,462 id=9bd3899c0ab5976c ip=10.1.0.12 udp=54556 tcp=45153
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:10.483] New local node record                    seq=1,736,987,290,463 id=9bd3899c0ab5976c ip=10.1.0.12 udp=54556 tcp=45153
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-16T00:28:10.916Z	INFO	first_node	status-cli/util.go:68	account created, key UID: 0x0148026bc479d6f926ac9222d93ef7f803f6168ce556cb03176578a8e9df877e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:10.917] New local node record                    seq=1,736,987,290,464 id=9bd3899c0ab5976c ip=10.1.0.12 udp=54556 tcp=45153
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-16T00:28:10.950Z	INFO	first_node	status-cli/util.go:93	messenger started, public key: 0x04be3d0600b031d36f76f0fcaac0a1621ca7a417aee7bdb644a1ae5327a1e7c2d9dfaa26c82189875fd84934851539f0f5445a8049cb33b4efb50f18ff4c25cc47
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-16|00:28:15.477] New local node record                    seq=1,736,987,290,465 id=9bd3899c0ab5976c ip=20.161.78.96 udp=54556 tcp=50313
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-16T00:28:15.950Z	INFO	first_node	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131', '-kid', '0x318e25b042b429ec764e0258b3e34525b6d0ffaa13d6dd0a68ba3d5cc8c11392']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - 2025-01-16T00:28:16.242Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - -name 0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - -key-uid 0x318e25b042b429ec764e0258b3e34525b6d0ffaa13d6dd0a68ba3d5cc8c11392
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - 2025-01-16T00:28:16.242Z	INFO	0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:16.274] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - ERROR[01-16|00:28:16.279] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:16.280] HTTP server started                      endpoint=127.0.0.1:6131 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:16.279] New local node record                    seq=1,736,315,414,675 id=461975f0c72cbc16 ip=127.0.0.1 udp=0 tcp=45697
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:16.283] Started P2P networking                   self="enode://ee83895eaae78bb7bbfc7067bf8e53d77602b13bbebd4576aa44541a3718d2ca08edc5fb2534cbdc4414bcb93361ac271101867188c94b5fe79ad4610bab2c05@127.0.0.1:45697?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:16.303] New local node record                    seq=1,736,987,296,284 id=d4719baf88de8262 ip=10.1.0.12 udp=0 tcp=33619
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:16.304] New local node record                    seq=1,736,987,296,285 id=d4719baf88de8262 ip=10.1.0.12 udp=58506 tcp=33619
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:16.306] New local node record                    seq=1,736,987,296,286 id=d4719baf88de8262 ip=10.1.0.12 udp=58506 tcp=33619
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - 2025-01-16T00:28:16.437Z	INFO	0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131	status-cli/util.go:62	existing account, key UID: 0x318e25b042b429ec764e0258b3e34525b6d0ffaa13d6dd0a68ba3d5cc8c11392
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:16.438] New local node record                    seq=1,736,987,296,287 id=d4719baf88de8262 ip=10.1.0.12 udp=58506 tcp=33619
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - 2025-01-16T00:28:16.461Z	INFO	0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131	status-cli/util.go:93	messenger started, public key: 0x042bcad344d9b1497d7beb6a43e75d89a7890adc0f0ff382548681514ecbb2bf95e8b79fc2fe20d8ce84f1b0ba6be71ae16eb8f1c467fab7bacd08343d8dd1d44a
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138', '-kid', '0x601f5d3cdb4e5b88c3ff16f048ae40007d6286156316ca1c672630689bf9921a']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - 2025-01-16T00:28:18.243Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - -name 0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - -key-uid 0x601f5d3cdb4e5b88c3ff16f048ae40007d6286156316ca1c672630689bf9921a
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - 2025-01-16T00:28:18.243Z	INFO	0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:18.272] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - ERROR[01-16|00:28:18.278] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:18.279] HTTP server started                      endpoint=127.0.0.1:6138 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:18.278] New local node record                    seq=1,736,315,478,560 id=f7f3920bc8a72e6b ip=127.0.0.1 udp=0 tcp=40587
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:18.280] Started P2P networking                   self="enode://3ac5ea424c2432f8abea83f2bda40eb6a9d547a3a13689037567b082b6294f69992e815498c4bf31b64552d0966076c88f979f25f2d9229dab39676753262ac8@127.0.0.1:40587?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:18.301] New local node record                    seq=1,736,987,298,285 id=3183714e1bd737b2 ip=10.1.0.12 udp=0 tcp=37799
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:18.303] New local node record                    seq=1,736,987,298,286 id=3183714e1bd737b2 ip=10.1.0.12 udp=48626 tcp=37799
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:18.305] New local node record                    seq=1,736,987,298,287 id=3183714e1bd737b2 ip=10.1.0.12 udp=48626 tcp=37799
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - 2025-01-16T00:28:18.441Z	INFO	0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138	status-cli/util.go:62	existing account, key UID: 0x601f5d3cdb4e5b88c3ff16f048ae40007d6286156316ca1c672630689bf9921a
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:18.442] New local node record                    seq=1,736,987,298,288 id=3183714e1bd737b2 ip=10.1.0.12 udp=48626 tcp=37799
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - 2025-01-16T00:28:18.466Z	INFO	0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138	status-cli/util.go:93	messenger started, public key: 0x04b1b3dd79a7c6ddc51eb5cd008812c32509ffacc3283a2c94c4183287c34e936181bb9da951efe7ca39b3d8e54beaab1bd0d82faed7de3557ee07a3d0fbaa9080
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132', '-kid', '0x76b265c33f51e1e29b14c7a9e3e1043c624d5f9628abc7a37bb32326168a9a29']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - 2025-01-16T00:28:20.245Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - -name 0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - -key-uid 0x76b265c33f51e1e29b14c7a9e3e1043c624d5f9628abc7a37bb32326168a9a29
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - 2025-01-16T00:28:20.245Z	INFO	0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:20.275] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - ERROR[01-16|00:28:20.282] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:20.282] New local node record                    seq=1,736,315,423,817 id=232f620b5bea021f ip=127.0.0.1 udp=0 tcp=40255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:20.282] Started P2P networking                   self="enode://b3542774027c24d996420dff85eb139ae23bdcde18d453dbc3c73ffaf416adf50129aa503379fb51c568c61438d87b7857765de2b8b849dc0fc3db3f6457a335@127.0.0.1:40255?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:20.283] HTTP server started                      endpoint=127.0.0.1:6132 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:20.308] New local node record                    seq=1,736,987,300,290 id=8ef7bf802fc86b3f ip=10.1.0.12 udp=0 tcp=39265
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:20.310] New local node record                    seq=1,736,987,300,291 id=8ef7bf802fc86b3f ip=10.1.0.12 udp=39099 tcp=39265
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:20.312] New local node record                    seq=1,736,987,300,292 id=8ef7bf802fc86b3f ip=10.1.0.12 udp=39099 tcp=39265
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - 2025-01-16T00:28:20.449Z	INFO	0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132	status-cli/util.go:62	existing account, key UID: 0x76b265c33f51e1e29b14c7a9e3e1043c624d5f9628abc7a37bb32326168a9a29
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:20.449] New local node record                    seq=1,736,987,300,293 id=8ef7bf802fc86b3f ip=10.1.0.12 udp=39099 tcp=39265
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - 2025-01-16T00:28:20.468Z	INFO	0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132	status-cli/util.go:93	messenger started, public key: 0x047c462afdfee45ebea4494ac9913d2a615a015c5e5869443959f9e234c3a6ce77857e2878f7c4929ba6a9ba05bcfced693b6482a305f1a5e1ffd6d3a16bd343e5
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - INFO [01-16|00:28:21.300] New local node record                    seq=1,736,987,296,288 id=d4719baf88de8262 ip=20.161.78.96 udp=58506 tcp=50304
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - 2025-01-16T00:28:21.461Z	INFO	0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135', '-kid', '0x708a30bfc69bdd791e0554d9713b660435d7a8e345d566251689005c3c87ac19']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - 2025-01-16T00:28:22.246Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - -name 0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - -key-uid 0x708a30bfc69bdd791e0554d9713b660435d7a8e345d566251689005c3c87ac19
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - 2025-01-16T00:28:22.246Z	INFO	0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:22.280] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - ERROR[01-16|00:28:22.285] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:22.285] New local node record                    seq=1,736,315,451,186 id=c4a6f603a5f298f7 ip=127.0.0.1 udp=0 tcp=46373
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:22.285] Started P2P networking                   self="enode://c9b636283622d5569b588181599af7fa3c6e235be801e8fd89379129098d75b2af07b24ea20b223ab0a7ba6168a6442477bace4a227044e32d212495bed19cda@127.0.0.1:46373?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:22.286] HTTP server started                      endpoint=127.0.0.1:6135 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:22.307] New local node record                    seq=1,736,987,302,290 id=f177bb7bb68afbc8 ip=10.1.0.12 udp=0 tcp=41255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:22.308] New local node record                    seq=1,736,987,302,291 id=f177bb7bb68afbc8 ip=10.1.0.12 udp=46083 tcp=41255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:22.310] New local node record                    seq=1,736,987,302,292 id=f177bb7bb68afbc8 ip=10.1.0.12 udp=46083 tcp=41255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - 2025-01-16T00:28:22.447Z	INFO	0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135	status-cli/util.go:62	existing account, key UID: 0x708a30bfc69bdd791e0554d9713b660435d7a8e345d566251689005c3c87ac19
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:22.448] New local node record                    seq=1,736,987,302,293 id=f177bb7bb68afbc8 ip=10.1.0.12 udp=46083 tcp=41255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - 2025-01-16T00:28:22.472Z	INFO	0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135	status-cli/util.go:93	messenger started, public key: 0x048056281da125dfc429df85b69fcf13286a346faae25073a1dc0955d86f91661764d928b6ee59e0ae150a977d497c79fe069266e80890a3b106d4ec6ca8b3bc14
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - INFO [01-16|00:28:23.297] New local node record                    seq=1,736,987,298,289 id=3183714e1bd737b2 ip=20.161.78.96 udp=48626 tcp=50305
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - 2025-01-16T00:28:23.466Z	INFO	0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136', '-kid', '0x924a030ee7140e04e1f74fef2d6293d230a53f7a73f77c29ad6df870b0c26f58']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - 2025-01-16T00:28:24.252Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - -name 0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - -key-uid 0x924a030ee7140e04e1f74fef2d6293d230a53f7a73f77c29ad6df870b0c26f58
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - 2025-01-16T00:28:24.252Z	INFO	0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:24.285] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - ERROR[01-16|00:28:24.289] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:24.290] New local node record                    seq=1,736,315,460,313 id=0f414d769065d7f6 ip=127.0.0.1 udp=0 tcp=40181
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:24.290] Started P2P networking                   self="enode://85e0d6a952fbc7dd1220f73edbc39659c964d8be3a2a4d8f951f1bb019758fa87a78eb700fe780127b6d0e15ec2a9b723d1f8fb8441c74f48eb19871dd884220@127.0.0.1:40181?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:24.290] HTTP server started                      endpoint=127.0.0.1:6136 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:24.321] New local node record                    seq=1,736,987,304,300 id=e01d2177be4772de ip=10.1.0.12 udp=0 tcp=41591
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:24.323] New local node record                    seq=1,736,987,304,301 id=e01d2177be4772de ip=10.1.0.12 udp=57633 tcp=41591
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:24.325] New local node record                    seq=1,736,987,304,302 id=e01d2177be4772de ip=10.1.0.12 udp=57633 tcp=41591
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - 2025-01-16T00:28:24.460Z	INFO	0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136	status-cli/util.go:62	existing account, key UID: 0x924a030ee7140e04e1f74fef2d6293d230a53f7a73f77c29ad6df870b0c26f58
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:24.461] New local node record                    seq=1,736,987,304,303 id=e01d2177be4772de ip=10.1.0.12 udp=57633 tcp=41591
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - 2025-01-16T00:28:24.485Z	INFO	0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136	status-cli/util.go:93	messenger started, public key: 0x045c64ff62014879abe06ac69aedd119afe6a0dd7b37a813d8ed09cb36b04be0e2d4e6dfaff4a272676b5f9e2441638d1c9dbb6aa7eee0f81ea865628018bdab45
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - INFO [01-16|00:28:25.304] New local node record                    seq=1,736,987,300,294 id=8ef7bf802fc86b3f ip=20.161.78.96 udp=39099 tcp=50306
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - 2025-01-16T00:28:25.469Z	INFO	0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134', '-kid', '0x5d0a17a7274481d620bacfcb493efc223184541a71176880cd945de1a20e55e4']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - 2025-01-16T00:28:26.246Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - -name 0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - -key-uid 0x5d0a17a7274481d620bacfcb493efc223184541a71176880cd945de1a20e55e4
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - 2025-01-16T00:28:26.246Z	INFO	0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:26.277] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - ERROR[01-16|00:28:26.284] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:26.284] New local node record                    seq=1,736,315,442,058 id=b01a6054d8c74f08 ip=127.0.0.1 udp=0 tcp=34203
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:26.284] Started P2P networking                   self="enode://7033cd3427730b644b017c2de11010f24f9acdc444028e0e10a4cde4a83415096594264e4e7148521a04e3cca5ba9c7ea697361a60be9cd991d30d21cb58896f@127.0.0.1:34203?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:26.285] HTTP server started                      endpoint=127.0.0.1:6134 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:26.307] New local node record                    seq=1,736,987,306,289 id=c7be6345dedf11ba ip=10.1.0.12 udp=0 tcp=38327
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:26.309] New local node record                    seq=1,736,987,306,290 id=c7be6345dedf11ba ip=10.1.0.12 udp=32785 tcp=38327
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:26.313] New local node record                    seq=1,736,987,306,291 id=c7be6345dedf11ba ip=10.1.0.12 udp=32785 tcp=38327
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - 2025-01-16T00:28:26.450Z	INFO	0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134	status-cli/util.go:62	existing account, key UID: 0x5d0a17a7274481d620bacfcb493efc223184541a71176880cd945de1a20e55e4
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:26.450] New local node record                    seq=1,736,987,306,292 id=c7be6345dedf11ba ip=10.1.0.12 udp=32785 tcp=38327
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - 2025-01-16T00:28:26.470Z	INFO	0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134	status-cli/util.go:93	messenger started, public key: 0x048e51e7f5ef88ba97ed543053d8cea3fe3a8b4f10242680a4ab05a26bc78e4d665312b3145003835d5376c79f5da46cd5c7bc3a3472ca25e52a6ae5f96cf08189
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - INFO [01-16|00:28:27.303] New local node record                    seq=1,736,987,302,294 id=f177bb7bb68afbc8 ip=20.161.78.96 udp=46083 tcp=50308
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - 2025-01-16T00:28:27.472Z	INFO	0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - WARN [01-16|00:28:27.485] Served personal_ecRecover                reqid=1 duration="59.231µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - WARN [01-16|00:28:27.487] Served personal_ecRecover                reqid=2 duration="26.841µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - WARN [01-16|00:28:27.489] Served personal_ecRecover                reqid=3 duration="20.999µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - WARN [01-16|00:28:27.491] Served personal_ecRecover                reqid=4 duration="26.24µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - WARN [01-16|00:28:27.492] Served personal_ecRecover                reqid=5 duration="24.907µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027C4DAF49ABB525306FE58ECA0D83A292640F55B074BF273121D3CF36045609CE_6138 - WARN [01-16|00:28:27.494] Served personal_ecRecover                reqid=6 duration="26.97µs" err="signature must be 65 bytes long"
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137', '-kid', '0x798f0322faaab75637cc33ad2713eec5e6cae3c334b3132f326831c529ff1f1b']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - 2025-01-16T00:28:28.248Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - -name 0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - -key-uid 0x798f0322faaab75637cc33ad2713eec5e6cae3c334b3132f326831c529ff1f1b
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - 2025-01-16T00:28:28.248Z	INFO	0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:28.280] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - ERROR[01-16|00:28:28.288] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:28.288] New local node record                    seq=1,736,315,469,431 id=a03b19848aa8ae71 ip=127.0.0.1 udp=0 tcp=43417
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:28.288] Started P2P networking                   self="enode://47ec99314e9bd4433887524e8d1bb5cef991529c6327fd581e7092f7718939d40d4faa42e4196d56827662015e63c194bdfab39c442a8c5991d53d86f96e7944@127.0.0.1:43417?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:28.288] HTTP server started                      endpoint=127.0.0.1:6137 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:28.310] New local node record                    seq=1,736,987,308,292 id=33fdf0fa4d4c8869 ip=10.1.0.12 udp=0 tcp=34255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:28.311] New local node record                    seq=1,736,987,308,293 id=33fdf0fa4d4c8869 ip=10.1.0.12 udp=34541 tcp=34255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:28.318] New local node record                    seq=1,736,987,308,294 id=33fdf0fa4d4c8869 ip=10.1.0.12 udp=34541 tcp=34255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - 2025-01-16T00:28:28.454Z	INFO	0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137	status-cli/util.go:62	existing account, key UID: 0x798f0322faaab75637cc33ad2713eec5e6cae3c334b3132f326831c529ff1f1b
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:28.454] New local node record                    seq=1,736,987,308,295 id=33fdf0fa4d4c8869 ip=10.1.0.12 udp=34541 tcp=34255
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - 2025-01-16T00:28:28.479Z	INFO	0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137	status-cli/util.go:93	messenger started, public key: 0x040476b36b9b1e21e10d1f4692b77831000daab6e950e3f7945a72b143866d3fcb36546535a7ca0690cac29d228922ed111e5b3ec93c3da2db664bcc5c6c517536
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - INFO [01-16|00:28:29.318] New local node record                    seq=1,736,987,304,304 id=e01d2177be4772de ip=20.161.78.96 udp=57633 tcp=50309
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - 2025-01-16T00:28:29.487Z	INFO	0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - WARN [01-16|00:28:29.518] Served personal_ecRecover                reqid=1 duration="45.916µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - WARN [01-16|00:28:29.519] Served personal_ecRecover                reqid=2 duration="20.268µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - WARN [01-16|00:28:29.520] Served personal_ecRecover                reqid=3 duration="21.149µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - WARN [01-16|00:28:29.522] Served personal_ecRecover                reqid=4 duration="19.767µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - WARN [01-16|00:28:29.523] Served personal_ecRecover                reqid=5 duration="27.582µs" err="signature must be 65 bytes long"
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133', '-kid', '0xc6b69f738454549ebe9b59807d3b3503c2851546856bae718220827d11c9b27e']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - 2025-01-16T00:28:30.254Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - -name 0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - -key-uid 0xc6b69f738454549ebe9b59807d3b3503c2851546856bae718220827d11c9b27e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - 2025-01-16T00:28:30.254Z	INFO	0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - INFO [01-16|00:28:30.288] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - INFO [01-16|00:28:30.294] New local node record                    seq=1,736,315,432,934 id=05ef2615206d52de ip=127.0.0.1 udp=0 tcp=39225
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - INFO [01-16|00:28:30.294] Started P2P networking                   self="enode://ee17158108f7505794695af5548af97aa71823dc2595397bb6227242f669702d3502511f3c79a42b83171d53ff66ad61a9311848d7c91a0f09bc9bcab95f676e@127.0.0.1:39225?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - ERROR[01-16|00:28:30.295] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - INFO [01-16|00:28:30.295] HTTP server started                      endpoint=127.0.0.1:6133 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - INFO [01-16|00:28:30.323] New local node record                    seq=1,736,987,310,301 id=d7f4e81a37b2d013 ip=10.1.0.12 udp=47265 tcp=39387
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - INFO [01-16|00:28:30.325] New local node record                    seq=1,736,987,310,302 id=d7f4e81a37b2d013 ip=10.1.0.12 udp=47265 tcp=39387
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - WARN [01-16|00:28:30.464] Served personal_ecRecover                reqid=6 duration="41.758µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - 2025-01-16T00:28:30.472Z	INFO	0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133	status-cli/util.go:62	existing account, key UID: 0xc6b69f738454549ebe9b59807d3b3503c2851546856bae718220827d11c9b27e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - INFO [01-16|00:28:30.473] New local node record                    seq=1,736,987,310,303 id=d7f4e81a37b2d013 ip=10.1.0.12 udp=47265 tcp=39387
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - 2025-01-16T00:28:30.499Z	INFO	0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133	status-cli/util.go:93	messenger started, public key: 0x049e70b35da90477771dc072daad7f5d2a9e9c47242ee1fa00b8291ed90cc899e88f919e77d838966aa59c0f7c935c18519c019c990077c309ca7c9d24a8b3c401
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:28:31.304] New local node record                    seq=1,736,987,306,293 id=c7be6345dedf11ba ip=20.161.78.96 udp=32785 tcp=50310
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - 2025-01-16T00:28:31.470Z	INFO	0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139', '-kid', '0x74fe52cbce9a0f5b5f54a3a7de81c5e1d33d442b688c78cb0d9103e3737199e2']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:32.251Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - -name 0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - -key-uid 0x74fe52cbce9a0f5b5f54a3a7de81c5e1d33d442b688c78cb0d9103e3737199e2
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:32.251Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:32.285] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - ERROR[01-16|00:28:32.290] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:32.290] New local node record                    seq=1,736,315,487,685 id=6a9c74352e0bbe70 ip=127.0.0.1 udp=0 tcp=42785
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:32.290] Started P2P networking                   self="enode://9cc4d3c1df1a690a18d32400d61656479914198d14fc843cca7825d5afabc2b51698872e9f1224d153e63dce363fff5f99e0552968c8294ae01fe35c6f0af3c0@127.0.0.1:42785?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:32.291] HTTP server started                      endpoint=127.0.0.1:6139 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:32.319] New local node record                    seq=1,736,987,312,295 id=94f74dec8fb0ec3b ip=10.1.0.12 udp=0 tcp=34875
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:32.321] New local node record                    seq=1,736,987,312,296 id=94f74dec8fb0ec3b ip=10.1.0.12 udp=37033 tcp=34875
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:32.322] New local node record                    seq=1,736,987,312,297 id=94f74dec8fb0ec3b ip=10.1.0.12 udp=37033 tcp=34875
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:32.471Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/util.go:62	existing account, key UID: 0x74fe52cbce9a0f5b5f54a3a7de81c5e1d33d442b688c78cb0d9103e3737199e2
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:32.471] New local node record                    seq=1,736,987,312,298 id=94f74dec8fb0ec3b ip=10.1.0.12 udp=37033 tcp=34875
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:32.502Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/util.go:93	messenger started, public key: 0x047d0a4d5c33b400598215eccc7ec9425c3f1f7a06cc2a7bd6aa9620b68a12b42fc1a6ff5a90de2e932c8bf1dbb3da4f19b5c434c2cd090a6855803b01cb914892
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - INFO [01-16|00:28:33.307] New local node record                    seq=1,736,987,308,296 id=33fdf0fa4d4c8869 ip=20.161.78.96 udp=34541 tcp=50307
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - 2025-01-16T00:28:33.479Z	INFO	0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - WARN [01-16|00:28:33.501] Served personal_ecRecover                reqid=1 duration="121.187µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - WARN [01-16|00:28:33.503] Served personal_ecRecover                reqid=2 duration="55.394µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - WARN [01-16|00:28:33.505] Served personal_ecRecover                reqid=3 duration="52.639µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - WARN [01-16|00:28:33.507] Served personal_ecRecover                reqid=4 duration="56.175µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - WARN [01-16|00:28:33.512] Served personal_ecRecover                reqid=5 duration="53.68µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0256150046AF3C90E47D162C844A4D3E4C98BE2CA6A4FE3E8D63B996B1AFD6DD02_6136 - WARN [01-16|00:28:33.514] Served personal_ecRecover                reqid=6 duration="31.62µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - WARN [01-16|00:28:33.657] Served personal_ecRecover                reqid=1 duration="66.595µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - WARN [01-16|00:28:33.659] Served personal_ecRecover                reqid=2 duration="29.264µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - WARN [01-16|00:28:33.661] Served personal_ecRecover                reqid=3 duration="29.115µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - WARN [01-16|00:28:33.662] Served personal_ecRecover                reqid=4 duration="28.203µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - WARN [01-16|00:28:33.664] Served personal_ecRecover                reqid=5 duration="26.399µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - WARN [01-16|00:28:33.666] Served personal_ecRecover                reqid=6 duration="56.125µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 4 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - WARN [01-16|00:28:34.497] Served personal_ecRecover                reqid=1 duration="73.056µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - WARN [01-16|00:28:34.498] Served personal_ecRecover                reqid=2 duration="65.293µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - WARN [01-16|00:28:34.503] Served personal_ecRecover                reqid=3 duration="30.477µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - WARN [01-16|00:28:34.505] Served personal_ecRecover                reqid=4 duration="62.566µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - WARN [01-16|00:28:34.506] Served personal_ecRecover                reqid=5 duration="28.684µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0367A8C7DC84FBE094059C2F552078F4DAACA83048FA839D29E9E4195E1B16457F_6132 - WARN [01-16|00:28:34.508] Served personal_ecRecover                reqid=6 duration="57.948µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - INFO [01-16|00:28:35.319] New local node record                    seq=1,736,987,310,304 id=d7f4e81a37b2d013 ip=20.161.78.96 udp=47265 tcp=50307
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - 2025-01-16T00:28:35.501Z	INFO	0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.509] Served personal_ecRecover                reqid=1 duration="73.538µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.514] Served personal_ecRecover                reqid=2 duration="31.409µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.516] Served personal_ecRecover                reqid=3 duration="25.668µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.519] Served personal_ecRecover                reqid=4 duration="25.257µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.520] Served personal_ecRecover                reqid=5 duration="26.96µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.522] Served personal_ecRecover                reqid=6 duration="25.968µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.525] Served personal_ecRecover                reqid=7 duration="24.726µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.527] Served personal_ecRecover                reqid=8 duration="23.844µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.528] Served personal_ecRecover                reqid=9 duration="24.105µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - WARN [01-16|00:28:35.530] Served personal_ecRecover                reqid=10 duration="23.634µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - WARN [01-16|00:28:35.700] Served personal_ecRecover                reqid=1 duration="55.433µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - WARN [01-16|00:28:35.702] Served personal_ecRecover                reqid=2 duration="20.578µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - WARN [01-16|00:28:35.704] Served personal_ecRecover                reqid=3 duration="20.668µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - WARN [01-16|00:28:35.707] Served personal_ecRecover                reqid=4 duration="28.162µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - WARN [01-16|00:28:35.709] Served personal_ecRecover                reqid=5 duration="25.107µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CA408A7D4EFF12329F0370DD24E7738863D4649728898F520A4BD163D4B55F08_6135 - WARN [01-16|00:28:36.487] Served personal_ecRecover                reqid=6 duration="34.765µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - INFO [01-16|00:28:37.314] New local node record                    seq=1,736,987,312,299 id=94f74dec8fb0ec3b ip=20.161.78.96 udp=37033 tcp=50312
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:37.502Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 13010fe4-301f-4724-916c-2d7c15f6f3ff_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_24 (ID=0x096e68bb8a67aa4bd9fd6a663f22db6c9df2eecbe6fed39bb227492718723d15)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 179dedec-da41-4e16-b428-304db9688109_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_22 (ID=0xc94696f66d4e456f7085e70ebeed1f423a035dafad465b91241daeb12321bc8c)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: ffe60365-d67b-4267-9048-674d85ae7b1e_message_from_first_node_21 (ID=0xa45c12193cb8c3f1d96a5904c6c33d045d44b0adc89c6f2f9f5286f21c4d1bae)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 64562295-6db0-4b54-ae58-dcfb5b7eb2d7_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_18 (ID=0x5732b68c651334e869c45824c4de41b1aa765ec9a1943fbd1387ba2daa8c86b5)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 259abd29-0da4-44b0-aaa2-34bd7644bb53_message_from_first_node_15 (ID=0xaa118c2b2f7dced3b7548a4b4e8d27abef60cebb01de2b4e00d9ea231d0d122d)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 1fe8b37e-53a8-420a-a1eb-cc9cba7dc3e6_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_8 (ID=0x2c72380aee3718d5c82981812447352eae6d9ad7942c46da83384fcde981f91a)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 7ae431ba-1115-4f2d-b21a-942bbf8080f2_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_10 (ID=0x2181c295f3f9749d43fe131c39b769f72d43c005e6929484b05ea141f5644819)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 972e7b8d-ad76-4f7b-9f62-a66c3cdb710f_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_6 (ID=0xa8dc2f58043f7a4b94ca2fe4a73270eeab7400ff67d45e6175e3c4cc0f285ec0)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: c13a18f7-88cc-4bfa-956d-928d22e7a221_message_from_first_node_1 (ID=0xe03f29dc3d9c85b0854e3fa46e81be360e1499c3da6dd2ddedf94990aa308863)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 8217a73f-6077-41bf-9c81-8866bdd19430_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_0 (ID=0x6bb42d4fc584e07d366d5a18a82edae2a8e2d68e65e2e1590f409a10e1f8efa2)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 2f254bc6-b81e-4646-8fe9-c078f5241419_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_20 (ID=0x0bf3a956625ecc5a73854d5c0a2979b2e0058eec6b8a397e05825b8b25007970)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 8345de8f-a327-4e7b-96f1-51ca3ed463a7_message_from_first_node_19 (ID=0x96d3bcff97d6c68fe19228e6c4f3aa5a2a6557e102f40b2f87d1aec1f4c88e8b)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: d658b42b-78d6-4c08-8037-7bca48d46d0a_message_from_first_node_17 (ID=0x1f03dc394e31411a6a4a0af86a76cb4111237caf11c6be8163a10951c88c032f)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: e6ec8ff7-1d02-499e-8b2c-b1ccb47f9ff0_message_from_first_node_13 (ID=0x0ba5dcc395e8b4b9b9631b2518da2a812d320de8ff01982c7705e1347bccaa86)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 7b106686-7432-41c7-8a10-ee2650e37456_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_14 (ID=0xa65f58236e1de35974d0d59d674fc1e26a0b7a360a9541d45837df8d0ed431f5)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: f0290bfd-2ff3-45c6-bf5b-e8bc5d862a45_message_from_first_node_7 (ID=0x23e6ef89f5a0148aa8a42cab4ba65974cf68d7d5298b9e5d44748283f8f19cb8)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 97b54d32-7e9c-4d84-bbb2-1d3a3867d40d_message_from_first_node_5 (ID=0x0eda7e0ae24fdc671f1a5676391f0b4ab069ff6407da8d698f9bb1e3ec9e3b94)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 1c916903-6da4-42fe-86c8-7d571fb46e05_message_from_first_node_3 (ID=0xa8022ac3033adf69ad01623c76d6c0ef9ca90e0fdd24502bc39ca442362d7303)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 013f314a-fdc0-4c9b-84e2-2b3b39601aa7 (ID=0x653ba0eda5018f9dff50d4a2d7f00c86c9bff2daf03561651b925b3c4ab6d28d)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 630b86d9-1966-463b-a5fc-d37aa6ad94e0_message_from_first_node_9 (ID=0x07841db2a55998ee4174ab16aad62151d4febaaf28c26c3b1836a11fd18ef15f)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 1283f243-2afd-4473-aa1a-2fd897bde429 (ID=0xe0cfb7c27941107b48b9d51de36eeec9c414dc185be3b661a59a33f2194ab07d)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: eecec25b-31a1-4237-9d53-1cdec14b7e51_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_12 (ID=0x1d51ee54c5d4f18a56217dcec8b1caf41c5117f6ca81a5dfc40053df56b13965)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 24818249-bd58-482e-9bcf-b058604966cd_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_4 (ID=0x030feee1e6873281ca1b3ad7c5a4b41cda11527c65096b4c45bd2fbb65112245)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: b2ea4038-8318-4ca0-a7c3-b5eca5a12c71_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_2 (ID=0x845d284ac067b81f062a5c76a76b9278ce90c5b5faff1b67dde5d0e1a58cd2e2)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: ce258782-06d6-45e8-8b20-3acb9509f605_message_from_first_node_23 (ID=0xcd9e9c66c8ab006132b3141868bf40de7a2854bd8b2b8f5ac798cf06a3e5b0b4)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 3f3e5379-bce9-4cbe-9b24-235ea73fe8e6_message_from_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139_16 (ID=0xc092133b21559451592c8f9cb0a70be703971f890d9ed91cb81f48033b077af1)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022E34EA7105F4A5AD44CBC7094B2F7AD4FC29200F05FDC4E473F90F4CD41CA73C_6139 - 2025-01-16T00:28:38.195Z	INFO	0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139	status-cli/serve.go:70	message received: 90481eeb-8183-4d9f-9cff-26b155548885_message_from_first_node_11 (ID=0x7df2c9477e9f309f7f3a704ea87ad67b8adfb4d160b37030d0aac8dde1f55ad7)
[35mDEBUG   [0m src.steps.common:common.py:39 Entering context manager: add_latency
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:42339 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:25 Received response: {"jsonrpc":"2.0","id":1,"result":{"id":"0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b","memberRole":0,"isControlNode":false,"verified":false,"joined":false,"joinedAt":0,"spectated":false,"requestedAccessAt":0,"name":"vac_qa_community_1","description":"vac_qa_community_1","introMessage":"","outroMessage":"","tags":[],"chats":{"0f2a46f0-2472-407c-b71c-b493209ba192":{"id":"0f2a46f0-2472-407c-b71c-b493209ba192","name":"general","color":"#ffffff","emoji":"","description":"General channel for the community","members":null,"permissions":{"access":1},"canPost":false,"canView":false,"canPostReactions":false,"viewersCanPostReactions":false,"position":0,"categoryID":"","tokenGated":false,"hideIfPermissionsNotMet":false,"missingEncryptionKey":false}},"categories":{},"images":null,"permissions":{"access":3},"members":{"0x042bcad344d9b1497d7beb6a43e75d89a7890adc0f0ff382548681514ecbb2bf95e8b79fc2fe20d8ce84f1b0ba6be71ae16eb8f1c467fab7bacd08343d8dd1d44a":{"roles":[1]},"0x042c85723b0417e20b37df497a8bc9610455bb1d2d567101d0c46ae3cdace0121784ca1c7530e111ffc7341463a3fdb9bf89bf1c34cb2208d2365d138ff022f636":{"last_update_clock":1736987273},"0x042e5262fe2e7925f2e93fa715908bf60acb97347b7d729db845455a98eb1b9b536cfdb9534c5e44430ec40dce2833505f0aa16fbd338aea7d2ce530c36ba7f3d4":{"last_update_clock":1736901140},"0x044362c6b2709a63251f748e14f3fde779c82fdd0b889282678e2f0e2e00c9e459f52216202ca4c7a6289fb2cf72b290d0651154380cf11cc21b1e608889fd5d73":{"last_update_clock":1736900911},"0x048f8daa068f29df2ed945f6a38d072cafa3ab0cbfe763098d79e03b7f59c91fa8ad1e4f6789b5f4c4ad3624b3b3b5e908979064996c01c03ee6b8f6e92b40a23d":{"last_update_clock":1736900864},"0x04cbc4cc8a115003488eed317c9752519b13480be79ccb88f58f65300154b026f144904818eeec6db16391f5007c0313f11c5d27385f59ea2428e47cd7467704a7":{"last_update_clock":1736900976},"0x04ef5f7ed41925c3b2364078e0c70ee1a15c6c3bda1c7fa6f10005f18618d38933e0732dfa21857e08bf641f576ed88546f971f6482d60bb7f58fbbfa650ca8526":{"last_update_clock":1736901042}},"canRequestAccess":true,"canManageUsers":false,"canDeleteMessageForEveryone":false,"canJoin":false,"color":"#ffffff","isMember":false,"muted":false,"muteTill":"0001-01-01T00:00:00Z","adminSettings":{"pinMessageAllMembersEnabled":false},"encrypted":false,"pendingAndBannedMembers":{},"tokenPermissions":{},"communityTokensMetadata":null,"activeMembersCount":0,"pubsubTopic":"","pubsubTopicKey":"","shard":null,"lastOpenedAt":0,"clock":1736987288000}}

[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:42339 with payload: {"jsonrpc": "2.0", "method": "wakuext_requestToJoinCommunity", "params": [{"communityId": "0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b", "addressesToReveal": ["fakeaddress"], "airdropAddress": "fakeaddress"}], "id": 1}
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:25 Received response: {"jsonrpc":"2.0","id":1,"result":{"requestsToJoinCommunity":[{"id":"0x249ae910b2677e03a9084bb45e533a35818bc4e7bc955d886b100b5f36b6ed88","publicKey":"0x04be3d0600b031d36f76f0fcaac0a1621ca7a417aee7bdb644a1ae5327a1e7c2d9dfaa26c82189875fd84934851539f0f5445a8049cb33b4efb50f18ff4c25cc47","clock":1736987321,"chatId":"","communityId":"0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b","state":1,"our":true,"deleted":false,"revealedAccounts":[{"address":"fakeaddress","chain_ids":[1,10,42161],"isAirdropAddress":true}],"customizationColor":"#ffffff","shareFutureAddresses":false}],"notifications":null,"communities":[{"id":"0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b","memberRole":0,"isControlNode":false,"verified":false,"joined":false,"joinedAt":0,"spectated":false,"requestedAccessAt":0,"name":"vac_qa_community_1","description":"vac_qa_community_1","introMessage":"","outroMessage":"","tags":[],"chats":{"0f2a46f0-2472-407c-b71c-b493209ba192":{"id":"0f2a46f0-2472-407c-b71c-b493209ba192","name":"general","color":"#ffffff","emoji":"","description":"General channel for the community","members":null,"permissions":{"access":1},"canPost":false,"canView":false,"canPostReactions":false,"viewersCanPostReactions":false,"position":0,"categoryID":"","tokenGated":false,"hideIfPermissionsNotMet":false,"missingEncryptionKey":false}},"categories":{},"images":null,"permissions":{"access":3},"members":{"0x042bcad344d9b1497d7beb6a43e75d89a7890adc0f0ff382548681514ecbb2bf95e8b79fc2fe20d8ce84f1b0ba6be71ae16eb8f1c467fab7bacd08343d8dd1d44a":{"roles":[1]},"0x042c85723b0417e20b37df497a8bc9610455bb1d2d567101d0c46ae3cdace0121784ca1c7530e111ffc7341463a3fdb9bf89bf1c34cb2208d2365d138ff022f636":{"last_update_clock":1736987273},"0x042e5262fe2e7925f2e93fa715908bf60acb97347b7d729db845455a98eb1b9b536cfdb9534c5e44430ec40dce2833505f0aa16fbd338aea7d2ce530c36ba7f3d4":{"last_update_clock":1736901140},"0x044362c6b2709a63251f748e14f3fde779c82fdd0b889282678e2f0e2e00c9e459f52216202ca4c7a6289fb2cf72b290d0651154380cf11cc21b1e608889fd5d73":{"last_update_clock":1736900911},"0x048f8daa068f29df2ed945f6a38d072cafa3ab0cbfe763098d79e03b7f59c91fa8ad1e4f6789b5f4c4ad3624b3b3b5e908979064996c01c03ee6b8f6e92b40a23d":{"last_update_clock":1736900864},"0x04cbc4cc8a115003488eed317c9752519b13480be79ccb88f58f65300154b026f144904818eeec6db16391f5007c0313f11c5d27385f59ea2428e47cd7467704a7":{"last_update_clock":1736900976},"0x04ef5f7ed41925c3b2364078e0c70ee1a15c6c3bda1c7fa6f10005f18618d38933e0732dfa21857e08bf641f576ed88546f971f6482d60bb7f58fbbfa650ca8526":{"last_update_clock":1736901042}},"canRequestAccess":true,"canManageUsers":false,"canDeleteMessageForEveryone":false,"canJoin":false,"color":"#ffffff","requestedToJoinAt":1736987321,"isMember":false,"muted":false,"muteTill":"0001-01-01T00:00:00Z","adminSettings":{"pinMessageAllMembersEnabled":false},"encrypted":false,"pendingAndBannedMembers":{},"tokenPermissions":{},"communityTokensMetadata":null,"activeMembersCount":0,"pubsubTopic":"","pubsubTopicKey":"","shard":null,"lastOpenedAt":0,"clock":1736987288000}],"activityCenterNotifications":[{"id":"0x249ae910b2677e03a9084bb45e533a35818bc4e7bc955d886b100b5f36b6ed88","chatId":"","communityId":"0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b","membershipStatus":1,"name":"","author":"","installationId":"","type":7,"lastMessage":null,"message":null,"replyMessage":null,"timestamp":1736987321696,"read":true,"dismissed":false,"deleted":false,"accepted":false,"contactVerificationStatus":0,"tokenData":null,"updatedAt":1736987321696,"albumMessages":null}],"activityCenterState":{"hasSeen":true,"updatedAt":0},"discordOldestMessageTimestamp":0}}

[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:42339 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0262A45B93D6B04E177D73FAE7F95F9289C880BCF7E824933CE2E055238012E98B_6131 - WARN [01-16|00:28:43.464] Served personal_ecRecover                reqid=7 duration="60.814µs" err="signature must be 65 bytes long"
[1m[31mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:42339 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:42339 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - WARN [01-16|00:28:58.483] Served personal_ecRecover                reqid=1 duration="63.879µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - WARN [01-16|00:28:58.484] Served personal_ecRecover                reqid=2 duration="24.346µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - WARN [01-16|00:28:58.485] Served personal_ecRecover                reqid=3 duration="25.778µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - WARN [01-16|00:28:58.487] Served personal_ecRecover                reqid=4 duration="26.519µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - WARN [01-16|00:28:58.488] Served personal_ecRecover                reqid=5 duration="20.489µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - WARN [01-16|00:28:58.747] Served wakuext_fetchCommunity            conn=127.0.0.1:55474 reqid=1 duration=17.047347s err="too many failed requests"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - WARN [01-16|00:28:58.747] Served wakuext_fetchCommunity            conn=127.0.0.1:46888 reqid=1 duration=6.935656595s err="too many failed requests"
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:25 Received response: {"jsonrpc":"2.0","id":1,"error":{"code":-32000,"message":"too many failed requests"}}

[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:42339 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - WARN [01-16|00:28:58.997] Served wakuext_fetchCommunity            conn=127.0.0.1:38886 reqid=1 duration=147.978385ms err="too many failed requests"
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:25 Received response: {"jsonrpc":"2.0","id":1,"error":{"code":-32000,"message":"too many failed requests"}}

[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:42339 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - WARN [01-16|00:28:59.247] Served wakuext_fetchCommunity            conn=127.0.0.1:38896 reqid=1 duration=147.66958ms err="too many failed requests"
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:25 Received response: {"jsonrpc":"2.0","id":1,"error":{"code":-32000,"message":"too many failed requests"}}

[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:42339 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X027E44751004D002D708CCD30F2847DA5333A92627516E540D643605C60579ADB0_6137 - WARN [01-16|00:28:59.482] Served personal_ecRecover                reqid=6 duration="38.813µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - WARN [01-16|00:29:01.502] Served personal_ecRecover                reqid=1 duration="68.378µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - WARN [01-16|00:29:01.504] Served personal_ecRecover                reqid=2 duration="33.382µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - WARN [01-16|00:29:01.505] Served personal_ecRecover                reqid=3 duration="24.466µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - WARN [01-16|00:29:01.506] Served personal_ecRecover                reqid=4 duration="20.348µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - WARN [01-16|00:29:02.502] Served personal_ecRecover                reqid=5 duration="33.061µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0380A412BE83772589A7AD5B47FC338483149386996F5ECD8F24E8B8AB5692FFE1_6133 - WARN [01-16|00:29:02.503] Served personal_ecRecover                reqid=6 duration="31.529µs" err="signature must be 65 bytes long"
[1m[31mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:42339 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:42339 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03D6BF5F2F19521F6149ADEF0ADF597655810E21EDB2A1C885BB1519E8EA7708FD_6134 - INFO [01-16|00:29:11.304] New local node record                    seq=1,736,987,306,294 id=c7be6345dedf11ba ip=20.161.78.96 udp=32785 tcp=50306
[1m[31mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:42339 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.steps.common:common.py:44 Exiting context manager: add_latency
[35mDEBUG   [0m tests.conftest:conftest.py:42 Running fixture teardown: save_node_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:45 Test failed, attempting to rename and move logs to the test_results folder
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131.log to test_results/testjoincommunity_test_join_community_with_latency_0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137.log to test_results/testjoincommunity_test_join_community_with_latency_0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136.log to test_results/testjoincommunity_test_join_community_with_latency_0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133.log to test_results/testjoincommunity_test_join_community_with_latency_0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135.log to test_results/testjoincommunity_test_join_community_with_latency_0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138.log to test_results/testjoincommunity_test_join_community_with_latency_0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134.log to test_results/testjoincommunity_test_join_community_with_latency_0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file first_node.log to test_results/testjoincommunity_test_join_community_with_latency_first_node.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132.log to test_results/testjoincommunity_test_join_community_with_latency_0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139.log to test_results/testjoincommunity_test_join_community_with_latency_0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139.log
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: first_node
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x0262a45b93d6b04e177d73fae7f95f9289c880bcf7e824933ce2e055238012e98b_6131
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x027c4daf49abb525306fe58eca0d83a292640f55b074bf273121d3cf36045609ce_6138
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x0367a8c7dc84fbe094059c2f552078f4daaca83048fa839d29e9e4195e1b16457f_6132
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02ca408a7d4eff12329f0370dd24e7738863d4649728898f520a4bd163d4b55f08_6135
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x0256150046af3c90e47d162c844a4d3e4c98be2ca6a4fe3e8d63b996b1afd6dd02_6136
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x03d6bf5f2f19521f6149adef0adf597655810e21edb2a1c885bb1519e8ea7708fd_6134
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x027e44751004d002d708ccd30f2847da5333a92627516e540d643605c60579adb0_6137
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x0380a412be83772589a7ad5b47fc338483149386996f5ecd8f24e8b8ab5692ffe1_6133
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x022e34ea7105f4a5ad44cbc7094b2f7ad4fc29200f05fdc4e473f90f4cd41ca73c_6139